components/common/FloatingRibbon.tsx

```diff
@@ -1,7 +1,8 @@
+import { ReactElement, ReactNode } from 'react'
 import { Button as BaseButton, Center } from '@chakra-ui/react'
-import { RESPONSIVE_PADDING } from './ContentBox'
+import { RESPONSIVE_PADDING } from '../lib/Definitions'
 
-export default function FloatingRibbon({ children }) {
+export default function FloatingRibbon({ children }: { children: ReactNode }): ReactElement {
```

Comment:
This is where I think we might be getting overly explicit in typing. I believe the TypeScript compiler can infer the return type of this, and certainly any reader can recognize this is a React component, so what does the explicit return type add?

I know there's a lint warning that prompted you to do this, but I suggest we just turn it off. It's even more aggressive than TypeScript itself and, in general, I don't think it's helpful. I can actually remember reading a defence of TypeScript from an expert that specifically called out the fact that you almost never have to explicitly specify return types as a benefit. I'm inclined to agree with that.

Reply:
I agree here as well fwiw - I think we should adjust our lint config to turn off the warning.

Reply:
Do we only want to turn off the lint warning for ReactElement? I'm assuming we would still want return types for more complicated objects when we get into some of the data work?

Reply:
@mwickett @davesteinberg Is this the rule we are talking about turning off? 

https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/explicit-function-return-type.md